src/main/java/org/apache/commons/configuration2/ImmutableConfiguration.java

```diff
@@ -62,6 +62,20 @@ public interface ImmutableConfiguration {
      */
     boolean containsKey(String key);
 
+    /**
+     * Returns true if this configuration contains one or more matches to this value. This operation stops at first
```

Comment:
Be consistent in your Javadoc comments. In this case, "contains" methods that return a boolean are "testing" for the presence of something, so you can start the comment with "Tests" like "Tests whether this or that happens".

Reply:
Thank you for the feedback. I have adjusted this.